stock_picking_cancel_reason/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 ProThai Technology Co.,Ltd. (http://prothaitechnology.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Picking Cancel Reason",
+    "version": "14.0.1.0.0",
+    "author": "Prothai, Odoo Community Association (OCA)",
+    "category": "Stock",
+    "license": "AGPL-3",
+    "complexity": "normal",
```

Comment:
This should be easy on. I think this flag not very useful.
```suggestion
```

Reply:
Fixed.